rtf_table.cpp

```diff
@@ -69,10 +69,8 @@ std::string table::make()
    // first, we'll determine all the rowspans and leftsides
    for (row=begin(); row!=end(); ++row)
    {
-      if ((*row)->CellDefs->size()!=(*row)->Cells.size())
-         throw std::logic_error("Number of Cells and number of CellDefs are unequal!");
       for (cell_def=(*row)->CellDefs->begin(), cell=(*row)->Cells.begin(); 
-           cell!=(*row)->Cells.end();
+           cell!=(*row)->Cells.end() && cell_def!=(*row)->CellDefs->end();
```

Comment:
Yes, I have. I converted RTF specifaction (from here: https://www.microsoft.com/en-us/download/details.aspx?id=10725) to RTF format using LibreOffice Writer 5.1.4.2. Then I tried to make an HTML file from it using rtf2html. And I got that error